qml/js/functions.js

```diff
@@ -540,3 +540,7 @@ function getMessagesNeededForwardPermissions(messages) {
     }
     return neededPermissions
 }
+
+function isTablet(appWindow) {
```

Comment:
`isTablet()` sounds misleading, I'd suggest `isWideScreen()` instead

Reply:
I suggest isLargeLandscapeScreen @monich

Reply:
I'd also go for isWideScreen, but not really crucial for merging... ;)